python/vyos/ifconfig/bridge.py

```diff
@@ -274,20 +274,31 @@ def update(self, config):
         for member in (tmp or []):
             if member in interfaces():
                 self.del_port(member)
-        vlan_filter = 0
 
-        vlan_del = set()
-        vlan_add = set()
+        # enable/disable Vlan Filter
+        vlan_filter = 1 if 'enable_vlan' in config else 0
```

Comment:
Is there any reason 'vlan_filter' is not a string? We work with strings most of the time soe this should read: `'1' if 'enable_vlan' in config else '0'`, so you can also spare the cast to `str()` below!